app/views/categories/_form.html.erb

```diff
@@ -2,26 +2,24 @@
   <% if category.errors.any? %>
     <div style="color: red">
       <h2><%= pluralize(category.errors.count, "error") %> prohibited this category from being saved:</h2>
-
       <ul>
         <% category.errors.each do |error| %>
           <li><%= error.full_message %></li>
         <% end %>
       </ul>
     </div>
   <% end %>
-
-  <div>
-    <%= form.label :name, style: "display: block" %>
-    <%= form.text_field :name %>
-  </div>
-
-  <div>
-    <%= form.label :icon, style: "display: block" %>
-    <%= form.text_field :icon %>
+  <div class="card w-50 mx-auto mt-6">
+    <div class="d-flex">
+      <span class="d-flex ml-3 mt-2 mb-1"><svg xmlns="http://www.w3.org/2000/svg" width="40" height="40" viewBox="0 0 24 24"><path d="M12 0c-6.627 0-12 5.373-12 12s5.373 12 12 12 12-5.373 12-12-5.373-12-12-12zm0 22c-3.123 0-5.914-1.441-7.749-3.69.259-.588.783-.995 1.867-1.246 2.244-.518 4.459-.981 3.393-2.945-3.155-5.82-.899-9.119 2.489-9.119 3.322 0 5.634 3.177 2.489 9.119-1.035 1.952 1.1 2.416 3.393 2.945 1.082.25 1.61.655 1.871 1.241-1.836 2.253-4.628 3.695-7.753 3.695z"/></svg>
+      </span>
+      <%= form.file_field :image, required: true, class:"input-file mt-3 ml-3", placeholder: "Upload Image"%>
+    </div>
+    <div>
+      <%= form.text_field :name, required: true, placeholder: "Name", class: "form-control mt-4" %>
+    </div>
   </div>
-
-  <div>
-    <%= form.submit %>
+  <div class="mt-5">
+    <%= form.submit 'Save', class: "btn btn-primary" %>
   </div>
```

Comment:
You have made great improvements on your forms 👏🏾

Kindly improve your margins so that the forms are more visible on mobile view. Please make the form for adding new categories 100% width so that it is close to the design specification as possible.

![image](https://user-images.githubusercontent.com/38088818/215267888-7c2e28df-0e20-473b-8503-eea00da00318.png)